packages/image/src/components/runtime.scss

```diff
@@ -1,49 +1,62 @@
 @import "@concord-consortium/question-interactives-helpers/src/styles/helpers";
-.runtime{
-  height:100%;
-}
-.imageContainer{
-  width: 100%;
-  margin-bottom: 10px;
-  img{
-    max-width: 100%;
-    height: auto;
-  }
-  &.fitWidth{
-    img{
-      width: 100%;
+
+.runtime {
+  height: 100%;
+
+  .imageContainer {
+    width: 100%;
+    margin-bottom: 10px;
+
+    img {
+      max-width: 100%;
+      height: auto;
     }
-  }
-  &.originalDimensions{
-    img{
-      width: auto;
+
+    &.fitWidth {
+      img {
+        width: 100%;
+      }
+    }
+
+    &.originalDimensions {
+      img {
+        width: auto;
+      }
     }
   }
-}
-.caption{
-  font-size: 0.8em;
-  width: 100%;
-  line-height: 1.5em;
-}
 
-.credit{
-  font-size: 0.8em;
-  width: 100%;
-  line-height: 1.2em;
-}
+  .captionAndZoomIn {
+    display: flex;
+    flex-direction: row;
```

Comment:
Do we need to add `align-items: flex-start` here so the two columns (`textContainer` and `viewHighRes`) are aligned at the top or is that already taken care of in the default flex layout?

Reply:
It seems to work fine without it, so I guess it might be the default behavior already.
<img width="633" alt="Screenshot 2023-01-26 at 19 36 23" src="https://user-images.githubusercontent.com/767857/215158452-61a1e87d-29c4-4e25-ad66-7bb637a4cd2e.png">
<img width="642" alt="Screenshot 2023-01-26 at 19 35 08" src="https://user-images.githubusercontent.com/767857/215158481-388e040f-9889-4bc1-baf4-732a2945d8fb.png">